src/components/UI/Text/Textarea.vue

```diff
@@ -194,10 +204,12 @@ export default {
     },
   },
   methods: {
-    inputChange(event) {
+    update() {
```

Comment:
What is being updated here?
You should name the method such that it's clear what action is it doing

Reply:
Also add one-liner docstring above the method name

Reply:
Text is being updated here. I will change it to UpdateChange.
will add comments too!